seL4-platforms/platforms.yml

```
@@ -75,6 +75,14 @@ platforms:
     march: rv64imac
     no_hw_test: true
 
+  STAR64:
+    arch: riscv
+    modes: [64]
+    smp: [64]
+    platform: star64
+    req: [star64]
+    march: rv64imafdc
```

For now we still need the old version

```suggestion
    march: rv64imac
```